src/init.cpp

```diff
@@ -118,7 +118,7 @@ static const char* FEE_ESTIMATES_FILENAME="fee_estimates.dat";
 // threads that should only be stopped after the main network-processing
 // threads have exited.
 //
-// Note that if running -daemon the parent process returns from AppInit2
+// Note that if running -daemon the parent process returns from AppInitMain()
```

Comment:
This is no longer true, by the way. Since #9010, the -daemon handling is in bitcoind.cpp, not in init.cpp.